microsetta_private_api/api/tests/test_api.py

```diff
@@ -1757,30 +1766,146 @@ def sign_data_consent(self):
     def sign_biospecimen_consent(self):
         """Checks biospecimen consent for a source and sings the consent"""
 
-        dummy_acct_id, source_resp = create_dummy_source(
+        dummy_acct_id, dummy_source_resp = create_dummy_source(
             "Bo", Source.SOURCE_TYPE_HUMAN, DUMMY_HUMAN_SOURCE,
             create_dummy_1=True)
 
         consent_status = self.client.get(
             '/api/accounts/%s/sources/%s/consent/%s' %
-            (dummy_acct_id, source_resp["source_id"], BIOSPECIMEN_CONSENT),
+            (dummy_acct_id, dummy_source_resp["source_id"],
+             BIOSPECIMEN_CONSENT),
             headers=self.dummy_auth)
 
         self.assertTrue(consent_status["result"])
 
-        CONSENT_ID_BIO = "6b1595a5-4003-4d0f-aa91-56947eaf2901"
+        consent_id = "b8245ca9-e5ba-4f8f-a84a-887c0d6a2233"
+        consent_data = {
+            "age_range": "18-plus",
+            "participant_name": "Bo",
+            "consent_type": ADULT_BIOSPECIMEN_CONSENT,
+            "consent_id": consent_id
+        }
+
+        response = self.client.post(
+            '/api/accounts/%s/sources/%s/consent/%s' %
+            (dummy_acct_id, dummy_source_resp["source_id"],
+             BIOSPECIMEN_CONSENT),
+            content_type='application/json',
+            data=json.dumps(consent_data),
+            headers=self.dummy_auth)
+
+        self.assertEquals(201, response.status_code)
 
-        consent_data = copy.deepcopy(DUMMY_HUMAN_SOURCE)
-        consent_data.update("consent_type", ADULT_BIOSPECIMEN_CONSENT)
-        consent_data.update("consent_id", CONSENT_ID_BIO)
+    def sign_data_consent_new_age_invalid(self):
```

Comment:
This method won't execute as a test unless prefixed with `test_`

```suggestion
    def test_sign_data_consent_new_age_invalid(self):
```